viscm/bezierbuilder/curve.py

```diff
@@ -0,0 +1,59 @@
+from math import factorial
+
+import numpy as np
+
+
+def binom(n, k):
+    """Re-implement `scipy.special.binom`.
+
+    Reimplementing it ourself lets us avoid pulling in a dependency scipy just for that
+    one function.
+
+    FIXME: `scipy` is already a dependency. Delete this?
```

Comment:
Via matplotlib or otherwise? Yes, fine to delete imo.

Reply:
It was explicitly set as a dependency in `setup.py`'s `setup(..., install_requires=...)` attribute before I started work on this repo, so I think this function and comment were written before that. Haven't actually looked in to the history to determine if that's true ;)